modules/mono/glue/runtime_interop.cpp

```diff
@@ -1116,6 +1116,9 @@ void godotsharp_array_make_read_only(Array *p_self) {
 }
 
 void godotsharp_array_set_typed(Array *p_self, uint32_t p_elem_type, const StringName *p_elem_class_name, const Ref<CSharpScript> *p_elem_script) {
+	ERR_FAIL_NULL(p_elem_class_name);
+	ERR_FAIL_NULL(p_elem_script);
```

Comment:
This is clearly meant to be optional so instead the call below should be changed to:
```cpp
p_self->set_typed(p_elem_type, elem_class_name, p_elem_script ? p_elem_script->ptr() : nullptr);
```